actioncable/test/server/configuration_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+require 'active_support/core_ext/array/grouping'
+
+class ActionCable::Server::ConfigurationTest < ActionCable::TestCase
+  setup do
+    @config = ActionCable::Server::Configuration.new(connection_class: ActionCable::Connection)
+  end
+
+  test "load namespaced channels" do
+    @config.channel_load_paths << File.join(FIXTURE_LOAD_PATH, 'channels')
```

Comment:
I think, we can simply stub `channel_paths` method to test this functionality. We don't need _real_ files.